covid_moonshot/analysis/plots.py

```diff
@@ -252,6 +253,41 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[int], runs_details: List[int]
+) -> plt.Figure:
+
+    complex_phases = [run.complex_phase for run in runs]
+    job_ids = [run.JOBID for run in runs_details]
+
+    std_dev_store = []
+    jobid_store = []
+
+    for i, run in enumerate(runs):
+
+        complex_gens = run.analysis.complex_phase.gens
+
+        std_dev = np.std([gen.free_energy.delta_f for gen in complex_gens])
+
+        if std_dev * _kT_kcal >= 1:  # 1kcal/mol
```

Comment:
Could you please un-hardcode the 1 kcal/mol, and instead have it as a function variable (of default 1)